pandas-stubs/_typing.pyi

```diff
@@ -229,12 +229,15 @@ AggFuncTypeFrame: TypeAlias = (
 )
 
 num: TypeAlias = complex
-SeriesAxisType: TypeAlias = Literal[
-    "index", 0
-]  # Restricted subset of _AxisType for series
-AxisTypeIndex: TypeAlias = Literal["index", 0]
-AxisTypeColumn: TypeAlias = Literal["columns", 1]
-AxisType: TypeAlias = AxisTypeIndex | AxisTypeColumn
+# SeriesAxisType: TypeAlias = Literal[
+#     "index", 0
+# ]  # Restricted subset of _AxisType for series
+AxisIndex: TypeAlias = Literal["index", 0]
+# AxisIndex: TypeAlias = Literal["index", 0]
+AxisColumn: TypeAlias = Literal["columns", 1]
+# AxisColumn: TypeAlias = Literal["columns", 1]
+Axis: TypeAlias = AxisIndex | AxisColumn | Literal["rows"]
```

Comment:
Let's remove `Literal["rows"]`  Even though it works, we don't want to encourage this undocumented behavior.